ArcGISRuntimeSDKQt_QMLSamples/Routing/RouteAroundBarriers/RouteAroundBarriers.qml

```diff
@@ -0,0 +1,356 @@
+// [WriteFile Name=RouteAroundBarriers, Category=Routing]
+// [Legal]
+// Copyright 2020 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.2
+import Esri.ArcGISExtras 1.1
+import Esri.ArcGISRuntime 100.8
+import QtQuick.Layouts 1.11
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url dataPath: System.userHomePath + "/ArcGIS/Runtime/Data/symbol"
+    readonly property url routeTaskUrl: "http://sampleserver6.arcgisonline.com/arcgis/rest/services/NetworkAnalysis/SanDiego/NAServer/Route"
+    readonly property int checkBoxPadding: 20
+    property var stopsList: []
+    property var barriersList: []
+    property var createAndDisplayRoute
+    property var routeParameters: null
+    property var directionListModel: null
+    property bool addStops: false
+    property bool addBarriers: false
+    property bool findBestSeq: false
+    property bool preserveFirstStp: false
+    property bool preserveLastStp: false
+
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            BasemapStreets {}
+
+            ViewpointCenter {
+                Point {
+                    x: -117.1750
+                    y: 32.727
+                    SpatialReference {
+                        wkid: 4326
+                    }
+                }
+                targetScale: 40000
+            }
+        }
+
+        GraphicsOverlay {
+            id: stopsOverlay
+        }
+
+        PictureMarkerSymbol {
+            id: pinSymbol
+            url: dataPath + "/orange_symbol.png"
+        }
+
+        GraphicsOverlay {
+            id: routeOverlay
+            renderer: SimpleRenderer {
```

Comment:
optional, but you don't need to explicitly define the property. renderer is a default property, so any of that type that is a child of GraphicsOverlay will automatically be assigned

```suggestion
            SimpleRenderer {
```